source/dofs/dof_handler_policy.cc

```diff
@@ -4334,11 +4358,16 @@ namespace internal
 
                 // first decompress the buffer
                 {
+#ifdef DEAL_II_WITH_ZLIB
+
                   boost::iostreams::filtering_ostream decompressing_stream;
                   decompressing_stream.push(boost::iostreams::gzip_decompressor());
                   decompressing_stream.push(boost::iostreams::back_inserter(decompressed_buffer));
 
-                  decompressing_stream.write(&buffer[i*max_size], max_size);
+                  decompressing_stream.write();
```

Comment:
That looks suspicious and doesn't compile on clang. I guess this change was not intentional?

Reply:
Oops.